README.md

```diff
@@ -1,3 +1,3 @@
-# ublox-gnss
+# ublox_gnss
```

Comment:
```suggestion
# UBlox GNSS
```
AFAIK, the convention for converting a README header to a package name is that it should be the same but all lower case and with spaces replaced with `_`.